react/src/lib/components/DeckGLMap/layers/axes/axesLayer.stories.tsx

```diff
@@ -41,6 +51,10 @@ function ColoredLabels(props: { labelColor: string }) {
             ...layerProps,
             labelColor: getRgba(props.labelColor) as RGBAColor,
         }),
+        new NorthArrow3DLayer({
```

Comment:
Suggest creating a separate story for North Arrow only, preferably under the north arrow directory. It's easier to isolate regressions when each layer has its own isolated story.